airbyte-webapp/src/components/JobItem/components/MainInfo.tsx

```diff
@@ -142,8 +144,14 @@ const MainInfo: React.FC<IProps> = ({
       </InfoCell>
       <InfoCell>
         {!shortInfo && isNotCompleted && (
-          <CancelButton secondary onClick={onCancelJob}>
-            <FormattedMessage id="form.cancel" />
+          <CancelButton
+            secondary
+            disabled={isLoading}
+            isLoading={isLoading}
+            wasActive={showFeedback}
+            onClick={onCancelJob}
+          >
+            {showFeedback ? <FormattedMessage id="form.canceling" /> : <FormattedMessage id="form.cancel" />}
```

Comment:
Better to change the value than to have duplicate component render logic
```suggestion
            <FormattedMessage id={showFeedback ? "form.canceling" : "form.cancel"} />}
```

Reply:
Agree, will fix it